addon/flash/object.js

```diff
@@ -61,5 +78,13 @@ export default Ember.Object.extend(Evented, {
     const destroyTimer = run.later(this, 'destroyMessage', timeout);
 
     set(this, 'timer', destroyTimer);
+
+    const defaultDisapearTimeout = get(this, 'disappearTimeout');
```

Comment:
`disappearMessage` will never run, because the flash object would have been destroyed by the time it gets there. You should move the logic into `destroyMessage` instead, above [L33](https://github.com/jrjohnson/ember-cli-flash/blob/animateremoval/addon/flash/object.js#L33) so that it can delay destruction